src/zenml/materializers/built_in_materializer.py

```diff
@@ -66,7 +72,7 @@ def load(self, data_type: Type[Any]) -> Any:
             )
         return contents
 
-    def save(self, data: Any) -> None:
+    def save(self, data: Union[bool, float, int, str]) -> None:
```

Comment:
Now that this is getting specified, why not also support dicts?

Reply:
dicts are handled by `BuiltInContainerMaterializer` below